src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetApprenticeships/GetApprenticeshipsHandlerTests.cs

```diff
@@ -1183,5 +1183,140 @@ public class GetApprenticeshipsHandlerTests
             Assert.AreEqual("Should_Be_Third", actual.Apprenticeships.ElementAt(1).ApprenticeLastName);
             Assert.AreEqual("Should_Be_First", actual.Apprenticeships.ElementAt(2).ApprenticeLastName);
         }
+
+        [Test, MoqAutoData]
+        public async Task And_Is_Download_Then_Returns_Apprenticeships(
+            GetApprenticeshipsRequest request,
+            ApprenticeshipDetails apprenticeshipDetails,
+            Mock<IAlertsMapper> alertsMapper,
+            [Frozen] Mock<IProviderCommitmentsDbContext> mockContext,
+            [Frozen] Mock<IMapper<Apprenticeship, ApprenticeshipDetails>> mockMapper)
+        {
+            //Arrange
+            var mapper = new ApprenticeshipToApprenticeshipDetailsMapper(alertsMapper.Object);
+
+            var apprenticeships = new List<Apprenticeship>
```

Comment:
I can see you're setting up the list to be out of sequence but you can do this still with autodata if you want to reduce your setup code.

Reply:
Resolving as there's a lot of moving around of this code in another branch.